src/containers/CTNavigationContainer/CTNavigationContainer.js

```diff
@@ -136,27 +147,27 @@ const CTNavigationContainer = () => {
           }}
         />
         <Tab.Screen
-          name="Home"
+          name={STACK_SCREENS.STARRED}
           component={HomeNavigator}
           options={{
-            tabBarLabel: 'Home',
+            tabBarLabel: 'Starred',
```

Comment:
Why not use `STACK_SCREENS.STARRED` here?

Reply:
That's a good point, the only argument I have against that is that all of the `STACK_SCREENS` variable is meant to be used as an enum -- we use that in other parts of our code to specify the value of the screen that we want to push onto the navigator. So I think the underlying value of this enum shouldn't neccesarily be tied to the display name of the screen on the tab -- especially if translations were to ever be added into the app.

What are your thoughts on this?